APIs/schemas/transporttype-response-schema.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
   "title": "Transport Type",
-  "description": "Transport type used by the Sender or Receiver in URN format",
+  "description": "URN base transport type used by the Sender or Receiver (i.e. with any subclassifications or versions removed)",
```

Comment:
Just 2p - I prefer either "URN base _for the_ transport type" (as in the RAML), or "Transport type URN base"